prov/efa/src/efa_domain.c

```diff
@@ -111,7 +112,7 @@ static struct fi_ops_domain efa_domain_ops = {
 	.cq_open = efa_cq_open,
 	.endpoint = efa_ep_open,
 	.scalable_ep = fi_no_scalable_ep,
-	.cntr_open = fi_no_cntr_open,
+	.cntr_open = efa_cntr_open,
```

Comment:
Does the `rdm_cntr_pingpong` test run successfully now? If it does, we should remove it from our fabtest exclude list.

Reply:
It still doesn't work if I remove if from the exclude list. I think the efa counter would still need to be set up from rxr in order for this test to pass.